### ISSUE TITLE
logAnalyzer - duplicate entries

### ISSUE THREAD

**Issue by KITT3000** [jtsage/FS_XMLTools#2]
this script could be improved to fix duplicate errors, sometimes the log will write 300 errors like:
`Error: Running LUA method 'packetReceived'`
making log browsing cumbersome and this script could isolate it to only 1 entry

**Comment by jtsage** [comment]
good call, yeah, I can do that.  Ideally, I want to provide a (configurable) traceback for log errors too.  For example, if the error encountered includes "LUA", it should probably print the last 5 or 10 lines before it (the first time it sees it) - that seems helpful

**Comment by KITT3000** [comment]
also a good idea

**Comment by jtsage** [comment]
So, this one (silently) has 2 modes now - if the log starts with GIANTS ENGINE, it assumes it's a game log.

If it does not, it's really just a regex for "Error" and "Warning" with line numbers.

If it does, it grabs any errors, and tries to guess at the surrounding context - then it prints them, suppressing exact duplicates (minus the initial timestamp)

New option added --startup / --no-startup (default is yes) will print or suppress the errors before the first timestamp - in practice, this is all the warnings about bad mod options or unzipped files.  

I'm going to commit it for now, and test it for a day or two before I tag a new release.
